phpstan.neon

```diff
@@ -1,6 +1,6 @@
 parameters:
-    ignoreErrors:
-        - identifier: missingType.iterableValue
+    checkMissingIterableValueType: false
```

Comment:
This needs reverting to the original to fix a phpstan warning

Reply:
yes, we're not running the phpstan checks but this will be re-added in the next PR